src/main/java/seedu/notus/ui/Formatter.java

```diff
@@ -115,7 +115,7 @@ public static String formatNote(String message, Note note) {
         String header = message.concat(note.getTitle() + " " + note.getTagsName());
 
         if (note.getPinned()) {
-            header = header.concat("| ISPIN: " + note.getPinnedString() + " ");
+            header = header.concat("| ISPIN: " + note.getPinned() + " ");
```

Comment:
Suggestion for the message to just displayed "pinned" or "unpinned" without the " | ISPIN: ". Likewise for archive, can display "Archived" for archive notes

Reply:
Ramana suggested both to be true false.

I think ill display pinned" or "unpinned and archived when its archive. But when its unarchive i wouldn't show it.